core/src/main/java/kafka/server/builders/ReplicaManagerBuilder.java

```diff
@@ -179,6 +179,7 @@ public ReplicaManager build() {
         if (metadataCache == null) throw new RuntimeException("You must set metadataCache");
         if (logDirFailureChannel == null) throw new RuntimeException("You must set logDirFailureChannel");
         if (alterPartitionManager == null) throw new RuntimeException("You must set alterIsrManager");
+        if (brokerTopicStats == null) brokerTopicStats = new BrokerTopicStats(config.remoteLogManagerConfig().enableRemoteStorageSystem());
```

Comment:
Initializing brokerTopicStats if null, with the initialized config in L176 above.